pkg/config/system_probe.go

```diff
@@ -165,6 +165,9 @@ func InitSystemProbeConfig(cfg Config) {
 
 	// service monitoring
 	cfg.BindEnvAndSetDefault(join(smNS, "enabled"), false, "DD_SYSTEM_PROBE_SERVICE_MONITORING_ENABLED")
+
+	// enable/disable use of root net namespace
+	cfg.BindEnvAndSetDefault(join(netNS, "disable_root_netns"), false)
```

Comment:
Can we flip it to an enable flag, with the default being true? I find it easier to read that way.

Reply:
Maybe name it `use_root_netns`?